x/profiles/types/models_params.go

```diff
@@ -16,12 +17,13 @@ const (
 
 // Default profile paramsModule
 var (
-	DefaultMinNicknameLength = sdk.NewInt(2)
-	DefaultMaxNicknameLength = sdk.NewInt(1000) // Longest name on earth count 954 chars
-	DefaultRegEx             = `^[A-Za-z0-9_]+$`
-	DefaultMinDTagLength     = sdk.NewInt(3)
-	DefaultMaxDTagLength     = sdk.NewInt(30)
-	DefaultMaxBioLength      = sdk.NewInt(1000)
+	DefaultMinNicknameLength      = sdk.NewInt(2)
+	DefaultMaxNicknameLength      = sdk.NewInt(1000) // Longest name on earth count 954 chars
+	DefaultRegEx                  = `^[A-Za-z0-9_]+$`
+	DefaultMinDTagLength          = sdk.NewInt(3)
+	DefaultMaxDTagLength          = sdk.NewInt(30)
+	DefaultMaxBioLength           = sdk.NewInt(1000)
+	DefaultAppLinksExpirationTime = time.Hour * 24 * 7 * 4 * 6 // This duration is equal to roughly 5.5 months time in minutes 241920
```

Comment:
I take the `DefaultUnbondingTime` param as an example:
https://github.com/cosmos/cosmos-sdk/blob/6ea204994443edc7841df4f0dc86ee5b4edc8377/x/staking/types/params.go#L21

Reply:
Since we can't sum decimals in time params calculation I've added an `ExpirationTimeCorrectionFactor` that I calculated as follow:
-  4382.91 hours are 182 days;
- `DefaultExpirationTime` were 168 days;
- 182 - 168 = 14 days = 20160 minutes;
- `DefaultAppLinksExpirationTime` + 20160 minutes = 262080 = 5.9835 Months.